nsq.h

```diff
@@ -66,9 +64,18 @@ struct NSQDConnection {
     void (*close_callback)(struct NSQDConnection *conn, void *arg);
     void (*msg_callback)(struct NSQDConnection *conn, struct NSQMessage *msg, void *arg);
     void *arg;
+    struct NSQDConnInfo *info;
     struct NSQDConnection *next;
 };
 
+struct NSQDConnInfo {
```

Comment:
after looking at this though, why can't we hang all these vars off `NSQDConnection`?